.ci/release.sh

```diff
@@ -23,12 +23,12 @@ echo $PATH
 java -version
 
 publishArg=''
-if [[ "$dry_run" == "true" ]] ; then
-    echo "--- Build and publish the release :package: (dry-run)"
-    publishArg='publishAllPublicationsToDryRunRepository'
-else
+if [[ "$dry_run" == "false" ]] ; then
```

Comment:
Here the condition must be reversed (or the if/else blocks swapped).

Reply:
I don't think so.

`dry-run==false` then do a release. Otherwise, dry-run is enabled.

This the way we can avoid surprises, such as:


```bash
dry_run='ups'


if [[ "$dry_run" == "true" ]] ; then
  publishArg='publishAllPublicationsToDryRunRepository'
else
  publishArg='publishToSonatype closeAndReleaseStagingRepository'
fi
...


It will run a release even when `dry_run` was not set to false but `ups`


I understand `dry-run` is a bit overkilling with a `false` validation but it's safer IMO